.github/workflows/test.yml

```diff
@@ -9,16 +9,17 @@ on:
 jobs:
   build:
     name: Continuous Testing
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.17]
+        go-version: [1.17, 1.18]
```

Comment:
We should standardize on 1.18 IMO.  Notation is on this version.  notation-core-go should be upgraded to the same.

Reply:
`notation-go` uses go 1.17, and this pipeline test for both 1.17 and 1.18.